uploadNeuroDB/minc_insertion.pl

```diff
@@ -534,15 +534,28 @@ =head2 Methods
 ################################################################
 ####### Get the $sessionID  ####################################
 ################################################################
-my ($sessionID) = NeuroDB::MRI::getSessionID(
-        $subjectIDsref, 
-        $studyInfo{'DateAcquired'},
-        \$dbh,
-        $subjectIDsref->{'subprojectID'},
-        $db
-
-   );
-
+my($sessionRef, $errMsg) = NeuroDB::MRI::getSession(
+    $subjectIDsref, 
+    $studyInfo{'DateAcquired'},
+    $dbh,
+    $db
+);
+$subjectIDsref->{'SessionID'} = $sessionRef->{'ID'};
```

Comment:
should also add the `ProjectID` and `SubprojectID` key to the `$subjectIDsref` otherwise, `identify_scan_db` does not have that information (the keys are only present in `$sessionRef ` after the call to `getSession`

Reply:
Done.